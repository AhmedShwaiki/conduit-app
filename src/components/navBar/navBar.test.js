import toJson from "enzyme-to-json";
import { shallow, mount } from "enzyme";
import { React } from "react";
import NavBar from "./navBar";
import { BrowserRouter as Router } from "react-router-dom";
it("renders correctly", () => {
  const wrapper = shallow(<NavBar />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
