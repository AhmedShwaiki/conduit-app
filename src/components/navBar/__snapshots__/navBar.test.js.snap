// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.nav>
  <styled.ul>
    <styled.li>
      <Styled(Link)
        to="/"
      >
        conduit
      </Styled(Link)>
    </styled.li>
    <styled.ul>
      <styled.li>
        <Styled(NavLink)
          to="/"
        >
          Home
        </Styled(NavLink)>
      </styled.li>
      <styled.li>
        <Styled(NavLink)
          to="/login"
        >
          Sign in
        </Styled(NavLink)>
      </styled.li>
      <styled.li>
        <Styled(NavLink)
          to="/register"
        >
          Sign up
        </Styled(NavLink)>
      </styled.li>
    </styled.ul>
  </styled.ul>
</styled.nav>
`;
